### PR TITLE
Outline today on date picker

### PR DESCRIPTION
**Pull request by VladimirLakodei**
Fix issues:
https://github.com/probablykasper/date-picker-svelte/issues/46

### CODE REVIEW

File: src/lib/DatePicker.svelte [hunk 1]
```diff
@@ -436,6 +441,10 @@
       background-color: transparent
     &.other-month span
       opacity: 0.4
+    &.today
```
Comment (probablykasper) [inline src/lib/DatePicker.svelte:444]: The `:hover` style conflicts with `.today` (aka, the hover style isn't applied)

**Review by probablykasper — CHANGES_REQUESTED** [review]
I wonder if it's possible to have a nice style without introducing new CSS variables

**Comment by VladimirLakodei** [comment]
@probablykasper Fixed

**Comment by probablykasper** [comment]
Thanks! I went and made it so only the background changes on hover, while the 2px border is kept

**Comment by VladimirLakodei** [comment]
Nice!

**Pull request merged**